AsyncDisplayKit/Layout/ASLayoutSpec.h

```diff
@@ -88,6 +90,23 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ * An ASLayoutSpec subclass that can wrap a ASLayoutable and calculates the layout of the child.
+ */
+@interface ASWrapperLayoutSpec : ASLayoutSpec
```

Comment:
Look into `examples/AsyncDisplayKitOverview/Sample/Node Containers/OverviewASTableNode.m` for an example.

Currently in `layoutSpecThatFits:` you only can return a layout spec. Previously if you would like to set a `height` and `width` to a `ASLayoutable` you had to set the `sizeRange`of it and wrap it in a `ASStaticLayoutSpec`. Now you theoretically would be able to just return the `ASLayoutable` itself as you can set the width and height directly. The problem as already mentioned is you cannot return a `ASDisplayNode` in `layoutSpecThatFits:` so you basically have to have some kind of wrapper spec for that.